code/obj/item.dm

```diff
@@ -1154,14 +1158,13 @@ ADMIN_INTERACT_PROCS(/obj/item, proc/admin_set_stack_amount)
 	return "It is \an [t] item."
 
 /obj/item/attack_hand(mob/user)
-	var/checkloc = src.loc
+	var/obj/item/checkloc = src.loc
 	while(checkloc && !istype(checkloc,/turf))
-		if (isliving(checkloc) && checkloc != user)
-			if(src in bible_contents)
-				break
-			else
-				return 0
-		checkloc = checkloc:loc
+		if(isliving(checkloc) && checkloc != user)
+			break
+		else
+			return 0
+		checkloc = checkloc.loc
```

Comment:
This isn't doing what you think it's doing. A better solution might be just skipping this check if the item is contained *directly* in a mob, rather than being in a container in a mob, since the intent is to prevent people stealing stuff out of other people's backpacks (I think??)